### ISSUE TITLE
What is the source for the test/train dataset?

### ISSUE THREAD

**Issue by marcbelmont** [Deevoluation/ALPR#3]
What is the source of the images in  `ALPR/Main Program/Test_car_images_dataset/` ?

> The classifier is trainied with 47605 images constituting images 36 classes

Where are those images?

**Comment by Saqhas** [comment]
We have made the dataset overself by collecting images from various sources. The Dataset is not uploaded here yet.

**Comment by marcbelmont** [comment]
Thanks for the update. Could you publish the algorithm accuracy on the test images?

**Comment by Saqhas** [comment]
The Algorithm reached an accuracy of 80% on the test set of 481 images.
The performance on videos is nearly 90 % but we are still in the process of benchmarking this accuracy to describe it better in terms of fps of the video, position of the car, speed of the car and many more.
Please contact for any more details.

**Comment by Subhraj07** [comment]
> We have made the dataset overself by collecting images from various sources. The Dataset is not uploaded here yet.

Could you please upload the training dataset?

**Comment by Subhraj07** [comment]
> > We have made the dataset overself by collecting images from various sources. The Dataset is not uploaded here yet.
> 
> Could you please upload the training dataset?

If the data is huge you can share through Drive or similar.

**Comment by Saqhas** [comment]
Yes sure.
I will upload it.
Sorry for the late reply
The dataset is not huge.
We are working on a new and improved repo with better documentations.
But if you want it urgently then, I will share it with you by today

**Comment by Saqhas** [comment]
Sorry for the delay. Here is the link to the dataset: https://drive.google.com/open?id=1OnqplHhTV3LjI8uMUWp0s6_uZ-EJ9trx

For any further help. Please feel free to contact

**Comment by Subhraj07** [comment]
> Sorry for the delay. Here is the link to the dataset: https://drive.google.com/open?id=1OnqplHhTV3LjI8uMUWp0s6_uZ-EJ9trx
> 
> For any further help. Please feel free to contact

Thanks Buddy... :)
